Resources/Prototypes/Reagents/botany.yml

```
@@ -220,15 +220,16 @@
         factor: 4
         conditions:
         - !type:MetabolizerTypeCondition
-          type: [ Avali ]
+          type: Avali
```

```suggestion
          type: [ Avali ]
```
```csharp
[DataField(required: true)]
public ProtoId<MetabolizerTypePrototype>[] Type = default!;
```
It's an array, so it always needs to be list syntax.